getting-started/gcp/main.tf

```diff
@@ -1,55 +1,72 @@
 provider "google" {
-  project = "terraform-project"
-  region  = "us-central1"
-  zone    = "us-central1-c"
+  credentials = file("<NAME>.json")
+
+  project = var.project
+  region  = var.region
+  zone    = var.zone
+}
+
+module "network" {
+  source  = "terraform-google-modules/network/google"
+  version = "1.1.0"
+
+  network_name = "terraform-vpc-network"
+  project_id   = var.project
+
+  subnets = [
+    {
+      subnet_name   = "subnet-01"
+      subnet_ip     = var.cidrs[0]
+      subnet_region = var.region
+    },
+    {
+      subnet_name   = "subnet-02"
+      subnet_ip     = var.cidrs[1]
+      subnet_region = var.region
+
+      subnet_private_access = "true"
+    },
+  ]
+
+  secondary_ranges = {
+    subnet-01 = []
+    subnet-02 = []
+  }
+}
+
+# Remove or #comment out:
```

Comment:
What does the code do if this line is commented out? It's unclear if this is an instruction for the student or a leftover comment from an auto-conversion tool.

Reply:
In the guide itself, I use that technique to show the user exactly which lines to replace, eg: https://learn.hashicorp.com/terraform/gcp/modules - I left it in the final so they can compare both techniques...but if it's confusing I can just get rid of the commented out ones.